tools/buildsys/src/args.rs

```diff
@@ -114,6 +115,12 @@ pub(crate) struct BuildPackageArgs {
     #[arg(long, env = "BUILDSYS_VARIANT_FLAVOR")]
     pub(crate) variant_flavor: String,
 
+    #[arg(long, env = "BUILDSYS_VERSION_BUILD")]
+    pub(crate) version_build: String,
+
+    #[arg(long, env = "BUILDSYS_VERSION_BUILD_TIMESTAMP")]
+    pub(crate) version_build_timestamp: String,
```

Comment:
Other args have set a bad precedent by not having doc comments. Can you add them here and say wheat these are being used for?

Reply:
sure thing!